components/script/script_thread.rs

```diff
@@ -3333,7 +3329,7 @@ impl ScriptThread {
             self.replace_surrogates,
             self.user_agent.clone(),
             self.player_context.clone(),
-            self.event_loop_waker.as_ref().map(|w| (*w).clone_box()),
+            None,
```

Comment:
Hrm. ☺

If we are always passing `None` here can we also remove this argument...or is it ever non-`None`?

Reply:
hmm, here it is always None and Window::new() is being called just in script_thread. 

To remove `event_loop_waker` from `Window`, I need to be a bit careful, as there are a few pub functions that are using Window's event_loop_waker and are being called in different places especially in webgl code that i am not familiar with(such as `webgl_chan` and `get_event_loop_waker`).

So far I haven't encountered a single palce where `get_event_loop_waker()` is not None. I will keep us updated here.